supereeg/simulate.py

```diff
@@ -6,7 +6,7 @@
 from sklearn import datasets
 from .brain import Brain
 
-def simulate_locations(n_elecs=10):
+def simulate_locations(n_elecs=10, random_seed=False):
```

Comment:
Let's set `random_seed` to `True` by default, and then set it to `False` for the tests (I assume the tests are why it's `False` by default here?)

Reply:
so this is misleading... it is random as default.  
maybe a better name would be `set_random_seed` default False